### ISSUE TITLE
OpenHexa toolbox v1

### ISSUE THREAD

**Issue by pvanliefland** [BLSQ/openhexa#18]
**TL;DR:** a Python library installed in your OpenHexa environment (notebooks and pipelines) with utilities to work with external systems (especially DHIS2) and perform common tasks.

## Rationale

OpenHexa is quite open regarding to what code runs on the platform: you can basically write any Python or R program you like, providing that the necessary libraries are installed on OpenHexa.

This flexibility is welcome, but OpenHexa is a health-centric data platform, and we want to provide ready-to-use utilities to perform common tasks :

- Interaction with external systems (especially DHIS2)
- Data formatting
- Common GIS operations

## Approach

A Python library, published on pypi, installed by default in notebooks and pipelines docker images.
